cves/kernel/CVE-2017-18360.yml

```diff
@@ -268,8 +294,13 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    So this could have been done by ltierally anyone who wanted to change their linux
```

Comment:
Perhaps don't start these answers with "so", might flow a little easier and sound more professional without it.

Reply:
fixed